backend/src/main/java/io/spring/cloud/samples/animalrescue/backend/domain/AnimalController.java

```diff
@@ -64,6 +75,11 @@ public Mono<Void> submitAdoptionRequest(
 	}
 
 	@PutMapping("/animals/{animalId}/adoption-requests/{adoptionRequestId}")
+	@Operation(
+		summary = "Pet adoption API",
+		description = "Update pet adoption requests.",
+		tags = {"pet adoption"}
```

Comment:
I noticed the tags were missing in the generated json. Do you happen to know if this was due to `springdoc` or perhaps missing functionality on our side?

Reply:
Good eye!  It looks like[ it's there in the docs generated in springdoc](https://gist.github.com/jrshaffe/22515c807fab0feb02b1687168fc2aa1#file-animal-rescue-from-springdoc-json-L26-L28).  I can take our generation logic.

Reply:
I logged a [bug](https://app.zenhub.com/workspaces/tanzu---api-lifecycle-5c929bca8dcdbd6b61379e7c/issues/gh/pivotal-cf/spring-cloud-gateway-k8s/3898) for our openapi route conversion